MapboxCoreNavigation/Constants.swift

```diff
@@ -157,3 +157,9 @@ public var RouteControllerNumberOfSecondsForRerouteFeedback: TimeInterval = 10
 public var RouteControllerOpportunisticReroutingInterval: TimeInterval = 120
 
 let FasterRouteFoundEvent = "navigation.fasterRoute"
+
+/**
+ Domain which all errors from delegate methods live under.
+ */
+public let MGLErrorDomain = "MGLErrorDomain"
```

Comment:
`MGLErrorDomain` is the name of the constant used by the map SDK. Our non-GL libraries use `MB` as the class prefix, not `MGL`.